### PR TITLE
feature/mobile toast sheet

### LINKED ISSUE #143: add async toast

### PR DESCRIPTION
**Pull request by Syncretik**
resolves #143

### CODE REVIEW

File: lib/store.ts [hunk 1]
```diff
@@ -18,3 +18,22 @@ export const useUserStore = create<UserStore>((set) => ({
...
+interface ToastType {
+  showToast: boolean;
+  message?: string;
+}
+
+interface ToastStore extends ToastType {
+  setShowToast: (values: ToastType) => void;
+}
+
+export const defaultToastValues = {
```
Comment (Riglanto) [inline lib/store.ts:31]: Export not needed here.

File: components/Toast.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import Image from "next/image";
+import { useState, useEffect } from "react";
+import Sheet from "react-modal-sheet";
+
+import { useToastStore } from "@/lib/store";
+
+export default function Toast() {
+  const [toastDetails, showToast, setShowToast] = useToastStore((state) => [
+    state,
```
Comment (Riglanto) [inline components/Toast.tsx:9]: image

This is a bit weird, either take state object or unpack its params, not both :P

**Review by Riglanto — COMMENTED** [review]
Probably auto hide would be nice? Not sure

Looks a bit weird:
image

Maybe it should be smaller on desktop? :thinking:

**Comment by gglucass** [comment]
I'm not a fan of autohide

But as discussed the shadow on desktop does need to look different. I'm ok with the size.

**Comment by Syncretik** [comment]
I tried, not sure if I can explicitly change that as it's baked into the package :( I can take another look

**Review by gglucass — APPROVED** [review]

**Pull request merged**